### PR TITLE
Updated Lottie, added contentMode option in initialiser

### PR DESCRIPTION
**Pull request by JordanDuncan**
Updated the example app.js to show how this works.

Also added an Xcode .gitignore to the /iphone directory from [Github's gitignores](https://github.com/github/gitignore/blob/master/Objective-C.gitignore).

### CODE REVIEW

**Comment by yozef** [comment]
awesome on `contentMode` :)

File: iphone/Classes/TiLottieVectorView.m [hunk 1]
```diff
@@ -10,13 +10,29 @@
...
 {
     if (animationView == nil) {
         id resource = [[self proxy] valueForKey:@"resource"];
+        id mode = [[self proxy] valueForKey:@"contentMode"];
         ENSURE_TYPE(resource, NSString);
+        ENSURE_TYPE(mode, NSString);
+        
+        animationView = [LOTAnimationView animationNamed:resource];
+        
+        if ([mode isEqual: @"aspectFit"]) {
```
Comment (hansemannn) [inline iphone/Classes/TiLottieVectorView.m]: I'd prefer to use constants here, so we don't have to use string-compares. I can add the required changes tomorrow, are you ok with that?

Comment (JordanDuncan) [reply iphone/Classes/TiLottieVectorView.m]: Aah, that's a much better idea yeah

**Comment by hansemannn** [comment]
Changed it online now, keeping the string-compar's for now, but recommeding the constants in the readme (like done with `TEXT_ALIGNMENT_CENTER` and `"center"` in the SDK. Please validate the changes once again, I'll do the FT tomorrow!

**Comment by hansemannn** [comment]
CR passed, PR approved!

**Comment by hansemannn** [comment]
One note: I'm in the process of moving this repo to https://github.com/m1ga/ti.animation, for a solid cross-platform module with a centralized API. So this changes (update library, add content mode) should either be contributed there again, or I'd pick it up for the initial PR (https://github.com/m1ga/ti.animation/pull/2).

**Comment by JordanDuncan** [comment]
I suppose I'd have to wait for your iOS code to be merged before
contributing mines so it may be easier to include this in your PR?

**Comment by hansemannn** [comment]
Yep, you're right. I just made some adjustments that include the latest Lottie 1.0.2 release (guess that's what you also used) as well as some migrations to the constants to be more flexible. I guess @m1ga will merge it soon, so we can go from there. This repo will then redirect to the centralized one.

**Pull request merged**